deploy/components/selfsigned-ca/kustomization.yaml

```diff
@@ -64,8 +64,8 @@ patchesJson6902:
       Name: apiserver
     patch: |-
       - op: add
-        path: /spec/template/spec/containers/1/volumeMounts
-        value: [{"name":"trust","mountPath":"/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem","readOnly":true,"subPath":"tls-ca-bundle.pem"}]
+        path: /spec/template/spec/containers/1/volumeMounts/-
+        value: {"name":"trust","mountPath":"/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem","readOnly":true,"subPath":"tls-ca-bundle.pem"}
```

Comment:
There are similar changes needed in a couple other places. I should have looked for those before posting!

Reply:
and actually this change only worked on my branch where volumeMounts already exists. Using only this change will break.